### ISSUE TITLE
JAMS logo?

### ISSUE THREAD

**Issue by justinsalamon** [marl/jams#140]
This is not of any importance, but I like making logos for things, and I like things having logos.

A while back I made one for JAMS (which I've been using on my website in the absence of an official logo), so thought I'd pitch it again here:

![possible jams logo](http://www.justinsalamon.com/uploads/4/3/9/4/4394963/8672107_orig.png)

Thoughts/comments?

**Comment by urinieto** [comment]
Looking great!
Would it look too bad if the M would be a rotated \Sigma like the one in
ISMIR?

**Comment by justinsalamon** [comment]
Do you mean something like this (sorry about quick hack):

![jams_logo_sigma](https://cloud.githubusercontent.com/assets/3009670/22264019/86ee8a3c-e244-11e6-9e0f-67613713197d.png)

I think this could also work, though I wonder if, given that the J is already in the shape of a musical note, adding another detail makes it a little on the nose?

**Comment by urinieto** [comment]
Yeah, like this! I like it: now it's more like a JAM between music and math
:D

**Comment by justinsalamon** [comment]
👍 
Any other OG care to chime in?
@bmcfee  @ejhumphrey @rabitt @jonforsyth

**Comment by bmcfee** [comment]
I think I prefer it with a normal M.  Is the J/note a unicode symbol?

**Comment by justinsalamon** [comment]
> Is the J/note a unicode symbol?

No, just a J + photoshopping.

**Comment by bmcfee** [comment]
Found a unicode-alike candidate: ```{𝆷AMS}```
